src/top-navigation/interfaces.ts

```diff
@@ -98,6 +99,7 @@ export namespace TopNavigationProps {
     type: 'menu-dropdown';
     description?: string;
     items: ButtonDropdownProps.Items;
+    expandableGroups?: boolean;
```

Comment:
Wouldn't it be better to be able to set this property for each group individually?

Reply:
That is not supported in the button-dropdown.